service/view_building_coordinator.hh

```diff
@@ -31,6 +34,46 @@ using view_name = std::pair<sstring, sstring>;
 using view_tasks = std::map<view_building_target, dht::token_range_vector>;
 using base_tasks = std::map<view_name, view_tasks>;
 struct vbc_tasks : public std::map<table_id, base_tasks> {};
-}
+
```

Comment:
Better move the vocabulary types to its own header.

Reply:
Is it really needed? 
There aren't that many types and all of them are only used by `view_building_coordinator` and `system_keyspace`